server/routers/juries.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express')
+const router = express.Router();
+const apiError = require('../entities/api-error');
+
+//router.use(authMiddleware());
+
+router.post('/add', async (req, res) =>{
```

Comment:
Rename to `/`, same with all routes. For update and delete you should replace them with `/:id`

Reply:
Ok, thank you for the guidance. 😄